axon-server-connector/src/main/java/org/axonframework/axonserver/connector/query/subscription/AxonServerSubscriptionQueryResult.java

```diff
@@ -84,16 +102,23 @@ public AxonServerSubscriptionQueryResult(final io.axoniq.axonserver.connector.qu
                     completeFlux(fluxSink, result.updates().getError().orElse(null));
                 }
             });
-        }).doOnError(e -> result.updates().close())
-          .map(subscriptionSerializer::deserialize);
```

Comment:
My gut tells me this exact deserialize invocation at the end was paramount to make this work in the past...however, the commit messages of this class don't help me to deduce whether that gut feeling is true.
So, I'm trusting in you that you've tested subscription queries to work as intended with these changes.

Reply:
Hmm it does change the semantics a little bit. Now, it will close the updates if a serialization error occurs. Before, it would ignore them. But I need the trace data in the message for this to work. What do you think? 
It's either deserializing the message here, or not tracing the subscription query handler

Reply:
After a verbal discussion, we agreed to keep the adjusted solution. We do not envision it to cause issues, but we aim to keep a focus on this spot whenever subscription query predicaments arise.